contrib/packaging/bootupd-static-grub-migration.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=bootupd static GRUB config migration
+Documentation=https://github.com/coreos/bootupd
+ConditionPathExists=!/boot/.bootupd-static-migration-complete
+RequiresMountsFor=/sysroot /boot
+# Only run after a successful bootloader update
```

Comment:
Are there any systems which would enable the static migration but *not* the update service?

If not, why wouldn't we just fold this functionality into the `bootloader-update.service` via

```
ExecStart=bootupctl migrate
ExecStart=bootupctl update
```

Or I guess just have `bootupctl update --migrate-if-needed` or so.

Reply:
We indeed want all the systems that do the migration to update their bootloader before.

We don't need to do the migration on systems which already have a static GRUB configs and we can check the state for that, so maybe this should indeed be folded into a single command and moved from a stamp file to a field in the state JSON.

If we merge both units then it also means that we have to careful when rolling this out as we won't be easily able to enable one or the other independently. But that's also kind of OK as I'm looking at doing that in F41 as well before the F42 release.